### PR TITLE
Add support for SKStoreReviewController for iOS 10.3+

### PR DESCRIPTION
**Pull request by EpicDraws**
Set useStoreReviewController to use the new review controller available
in iOS 10.3+

### CODE REVIEW

**Comment by EpicDraws** [comment]
Addresses #278

**Comment by weepy** [comment]
Is this working ? Quite interested in using it !

**Comment by alexfoxy** [comment]
any ideas if this is solid ?

**Comment by jigmelam** [comment]
good. Please check & merge soon.
Thanks so much!

File: iRate/iRate.m [hunk 1]
```diff
@@ -851,70 +851,73 @@ - (void)promptForRating
...
-
-            //rate action
-            [alert addAction:[UIAlertAction actionWithTitle:self.rateButtonLabel style:UIAlertActionStyleDefault handler:^(__unused UIAlertAction *action) {
-                [self didDismissAlert:alert withButtonAtIndex:0];
-            }]];
-
-            //cancel action
-            if ([self showCancelButton])
+        float iOSVersion = [[UIDevice currentDevice].systemVersion floatValue];
+        if(self.useStoreReviewController && iOSVersion >= __IPHONE_10_3) {
```
Comment (sidan5) [inline iRate/iRate.m]: @EpicDraws Would be smarter to check if respondToSelector instead of iOS version...

**Comment by cbess** [comment]
@nicklockwood Will this be merged into the codebase?

**Comment by Jules2010** [comment]
+1

**Pull request merged**